bindings/ios/Private/DelegateMEGAGlobalListener.mm

```diff
@@ -97,3 +97,24 @@
         });
     }
 }
+
+void DelegateMEGAGlobalListener::onEvent(mega::MegaApi *api, mega::MegaEvent *event) {
+    switch (event->getType()) {
+        case MegaEvent::EVENT_ACCOUNT_CONFIRMATION:            
+            if (listener != nil && [listener respondsToSelector:@selector(onEvent:event:)]) {
+                MegaEvent *tempEvent = NULL;
+                if(event) {
```

Comment:
FYI, you don't really need to check `event`, you always receive a valid instance, like the `MegaError` or the `MegaRequest` at `onRequestFinish()`